client/src/plus/common/index.tsx

```diff
@@ -31,3 +32,22 @@ export function DataError({ error }: { error: Error }) {
     </NoteCard>
   );
 }
+
+export function _getIconLabel(url) {
+  let category = docCategory({ pathname: url });
```

Comment:
Here we could reuse the `getCategoryFromUrl()` function, suggested above.

Reply:
Let's do this later. https://github.com/mdn/yari/issues/5637